serde-proto/protobuf/v1/common_types.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+import "google/protobuf/wrappers.proto";
+
+package v1.common_types;
```

Comment:
Separate common into specific protos if we need more kinds in the future
```suggestion
package v1.pb_time;
```

Reply:
Done.